pkg/drivers/kic/kic.go

```diff
@@ -202,11 +202,13 @@ func (d *Driver) GetSSHKeyPath() string {
 
 // GetURL returns ip of the container running kic control-panel
```

Comment:
Update the comment here, as it isn't returning an IP. Maybe specifically say it returns a URL to the Docker endpoint to Docker inside of the container? It is inside, isn't it?